test/Mono.Linker.Tests.Cases/DataFlow/MethodParametersDataFlow.cs

```diff
@@ -0,0 +1,197 @@
+﻿using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace Mono.Linker.Tests.Cases.DataFlow
+{
+	[SetupCSharpCompilerToUse ("csc")]
```

Comment:
You shouldn't need this anymore.  I made roslyn the default compiler a week or two ago.